prombench/docs/gke.md

```diff
@@ -43,7 +43,7 @@ export GRAFANA_ADMIN_PASSWORD=password
 export DOMAIN_NAME=prombench.prometheus.io # Can be set to any other custom domain or an empty string when not used with the Github integration.
 export OAUTH_TOKEN=<generated token from github or set to an empty string " ">
 export WH_SECRET=<github webhook secret>
-export GITHUB_ORG=prometheus
```

Comment:
Let's ask Saket as well, maybe they remember why it could stop working (or was always not working?)

Reply:
it feels it's set in CI: https://github.com/GoogleCloudPlatform/prometheus-test-infra/blob/707bfe0e4ec2c65ad61b6297ddb3f3fc22c1a1b9/.cloudbuild/prombench.yaml#L20C13-L21C70

Reply:
Can we double check this change is necessary?

Reply:
(we discussed this on 1:1 with @Adib234)

I think what's happening is that this documentation file is not changed on our fork as we don't change too many things to stay light on forking. It was fine to not change this, but we ran prombench benchmarks through GitHub comments ([exactly this CI scripts](https://github.com/GoogleCloudPlatform/prometheus-test-infra/blob/master/.cloudbuild/prombench.yaml)), thus we didn't bother to update local run docs. As a result we got a bit of confusion.

After discussion we decided to keep this change, but can we add comment e.g. with `# For GH integrated comment flow, those envvars will be changed for you https://github.com/GoogleCloudPlatform/prometheus-test-infra/blob/707bfe0e4ec2c65ad61b6297ddb3f3fc22c1a1b9/.cloudbuild/prombench.yaml#L20`